var/spack/repos/builtin/packages/py-tensorflow-probability/package.py

```diff
@@ -63,9 +62,8 @@ class PyTensorflowProbability(Package):
 
     variant("py-tensorflow", default=False, description="Build with TensorFlow support")
```

Comment:
This variant should really be named "tf" to match `pip install tensorflow-probability[tf]`

Reply:
fine with me